inst/shiny/modules/tab_nca/nca_results.R

```diff
@@ -75,25 +75,20 @@ nca_results_server <- function(id, pknca_data, res_nca, settings, ratio_table, g
         )
 
       # Add flaging column in the pivoted results
-      # ToDo(Gerardo): Once PKNCAoptions allow specification of adj.r.squared,
-      #                we can simplify this part by using the PKNCA object
-      rules <- settings()$flags
-      rule_thr <- lapply(rules, FUN =  \(x) x$threshold)
-      rule_pretty_names <- translate_terms(names(rules), "PKNCA", "PPTEST")
-      rule_msgs <- paste0(rule_pretty_names, c(" < ", " > ", " > ", " < "))
-
-      rules_applied <- sapply(rules, FUN =  \(x) x$is.checked)
-      params_applied <- translate_terms(names(rules), "PKNCA", "PPTEST")[rules_applied]
-      params_applied <- names(final_results)[formatters::var_labels(final_results)
-                                             %in% params_applied]
-
-      if (length(params_applied) > 0) {
+      applied_flags <- settings()$flags[sapply(settings()$flags, FUN =  \(x) x$is.checked)]
```

Comment:
suggestion: doesn't `purrr::keep` do the job?
```suggestion
      applied_flags <- purrr::keep(settings()$flags, \(x) x$is.checked)
```